vertexai/snippets/text_input_test.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package snippets
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestTextInput(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	buf := &bytes.Buffer{}
+	err := generateContentFromText(buf, tc.ProjectID)
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	fmt.Println(buf)
```

Comment:
The test shouldn't print output if it succeeds.

Perhaps you can check that one or more of the expected response field names (e.g. `Candidates`) shows up in the buffer? That seems like a reasonable thing to check to validate that you got a well formed response.

Reply:
Added a check here